.github/workflows/release.yml

```diff
@@ -16,7 +17,7 @@ env:
 
 jobs:
   release:
-    if: github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release/')
+    if: github.event_name == 'workflow_dispatch'
```

Comment:
Temporarily removing the automated trigger, will bring back one manual trigger has been verified.

Reply:
No, I was thinking to do a manual release. Then bring it back.